prow/scripts/cluster-integration/kyma-gke-compass-integration.sh

```diff
@@ -209,6 +209,31 @@ function installKyma() {
     --data "global.domainName=${DOMAIN}" \
     --data "global.loadBalancerIP=${GATEWAY_IP_ADDRESS}"
 
+  "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-config-map.sh" --name "application-operator-overrides" \
```

Comment:
Please consider creating single YAML override for the job - it would be easier to read and maintain.

Reply:
I merged new overrides into one. It will be easier to maintain